bindings/core/src/response.rs

```diff
@@ -105,6 +105,9 @@ pub enum Response {
     /// - [`SignEd25519`](crate::method::SecretManagerMethod::SignEd25519)
     Ed25519Signature(Ed25519SignatureDto),
     /// Response for:
+    /// - [`SignEvm`](crate::method::SecretManagerMethod::SignEvm)
+    EvmSignature { public_key: String, signature: String },
```

Comment:
I think we need `#[serde(rename_all = "camelCase")]` here for public_key

Reply:
Is it not enough that it's on the enum?

Reply:
No, unfortunately not, this only takes care of the variant name

Reply:
You're right, thanks

Reply:
Allowing myself to resolve as Thoralf is on holidays.